e2e/images_test.go

```diff
@@ -44,6 +63,16 @@ b-simple-app:latest           simple
 	})
 }
 
+func TestImageListQuiet(t *testing.T) {
```

Comment:
I think a unit test would be easier for this feature, mocking the bundle store to configure it the way you want.